DOM I Repurpose/js/index.js

```diff
@@ -0,0 +1,204 @@
+
+// DOM I Assignment 
+
+// ## Task 1: Create selectors to point your data into elements
+//  * Create selectors by using any of the DOM element's methods
+//  * Note that IDs have been used on all images.  Use the IDs to update src path content
+
+// let navItems = document.querySelectorAll('nav a');
+
+// navItems.forEach(function(element){
+//   navItems.setAttribute("innerHTML" ['nav'][element])
+// })
+
+let navItems = document.getElementsByTagName("a");
+navItems[0].innerHTML = (siteContent["nav"]["nav-item-1"])
+navItems[1].innerHTML = (siteContent["nav"]["nav-item-2"])
+navItems[2].innerHTML = (siteContent["nav"]["nav-item-3"])
+navItems[3].innerHTML = (siteContent["nav"]["nav-item-4"])
+navItems[4].innerHTML = (siteContent["nav"]["nav-item-5"])
+
+
+//new nav item and make font colors greens
+let navContainer = document.querySelector('nav');
+let newNavAnchor= document.createElement('a');
+
+newNavAnchor.innerHTML = "Lambda";
+navContainer.prepend(newNavAnchor);
+
+navItems[0].style.color = 'green';
+navItems[1].style.color = 'green';
+navItems[2].style.color = 'green';
+navItems[3].style.color = 'green';
+navItems[4].style.color = 'green';
+navItems[5].style.color = 'green'; 
+
+
+// 
+
+let mainCTA = document.querySelector('h1');
+mainCTA.innerHTML = (siteContent['cta']['h1']);
+
+let ctaButton = document.querySelector('button');
+ctaButton.innerHTML = (siteContent['cta']['button']);
+
+let ctaImage = document.getElementById('cta-img');
+ctaImage.setAttribute('src', siteContent['cta']['img-src']);
+
+
+let docH4 = document.getElementsByTagName('h4');
+
+docH4[0].innerHTML = (siteContent['main-content']['features-h4']);
+docH4[1].innerHTML = (siteContent['main-content']['about-h4']);
+docH4[2].innerHTML = (siteContent['main-content']['services-h4']);
+docH4[2].innerHTML = (siteContent['main-content']['product-h4']);
+docH4[3].innerHTML = (siteContent['main-content']['vision-h4']);
+
+
+
+let midImage = document.getElementById('middle-img');
+midImage.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+let mainContentP = document.getElementsByTagName('p');
+mainContentP[0].innerHTML = (siteContent['main-content']['features-content']);
+mainContentP[1].innerHTML = (siteContent['main-content']['about-content']);
+
+mainContentP[2].innerHTML = (siteContent['main-content']['product-content']);
+mainContentP[3].innerHTML = (siteContent['main-content']['vision-content']);
+
+//footer
+
+docH4[5].innerHTML = (siteContent['contact']['contact-h4']);
+
+mainContentP[5].innerHTML = (siteContent['contact']['address']);
+mainContentP[6].innerHTML = (siteContent['contact']['phone']);
+mainContentP[7].innerHTML = (siteContent['contact']['email']);
+
+//copyright
+
+mainContentP[8].innerHTML = (siteContent['footer']['copyright']);
+
+// DOM II Assignment
+
+// * Create 10 unique event listeners based on the reference file in the objective.  
+
+// 1. changes cta font to red on button click
```

Comment:
Thanks for using comments to call these out -- they're helpful for me, and informative for you or anyone else who wants to refer back to this :smiley: